### PR TITLE
Release divviup-api 0.0.27

### PR DESCRIPTION
**Pull request by tgeoghegan**

### CODE REVIEW

**Comment by tgeoghegan** [comment]
@jbr this bumps the version in Cargo.toml, but I'm happy to wait until the outstanding dependabot PRs are merged before cutting the 0.0.27 release.

**Review by divergentdave — APPROVED** [review]

**Comment by jbr** [comment]
I'll review those and merge within the next hour (afk at the moment)

**Comment by jbr** [comment]
This is good to go when #545 merges

**Pull request merged**
